site/prisma/migrations/20230817030641_init/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "rooms" (
+    "room_id" SERIAL NOT NULL,
+    "has_started" BOOLEAN NOT NULL,
+
```

Comment:
This isn't following the design that we're going off of; "has_started" is fine (instead of using current_question=0), but the table is missing "current_question"